### PR TITLE
chore: update readme and allow env file for local variables

### LINKED ISSUE #422: Testing: Create .env.local for secrets
Currently there's a lot of overhead to run tests locally due to a number of environment variables that are passed in CI and returning undefined when running locally.

At the moment there isn't a need to use a secret manager as the project should be integrated with Capsule soon and all secrets can just be put into an env file.

A temporary solution for now is to create an .env.local file that's saved in 1Password and shared between dev & test team so that tests can be run locally. 

- [ ] Create local env file with secrets
- [ ] Update Readme on how to set up and run tests locally

### PR DESCRIPTION
**Pull request by jtsang586**
# Related issues 🔗

Closes #422 

# Description ℹ️

- Updated Readme to allow people to run Cypress tests locally for the trading app
- Changed Cypress variable so it's easier to use different test wallet
- Added git ignore for local env file
- Added `cypress.env.json` file to 1Password so test team can run tests with shared secrets
- Tests passing locally with above file

### CODE REVIEW

**Review by dexturr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,6 +73,14 @@ Run `yarn nx run <my-app>-e2e:e2e` to execute the e2e tests with [cypress](https
 
 Run `nx test my-app` to execute the unit tests with [Jest](https://jestjs.io), or `nx affected:test` to execute just unit tests affected by a change. You can also use `--watch` with these test to run jest in watch mode, see [Jest executor](https://nx.dev/packages/jest/executors/jest) for all CLI flags.
 
+#### Note: below only needed for trading-e2e
```
Comment (mattrussell36) [inline README.md:76]: chore (blocking): Can we change this title to just read 'Trading app E2E tests`

File: README.md [hunk 2]
```diff
@@ -73,6 +73,14 @@ Run `yarn nx run <my-app>-e2e:e2e` to execute the e2e tests with [cypress](https
...
+
+To run tests locally using your own wallets you can add the following environment variables to `Cypress.json`
```
Comment (mattrussell36) [inline README.md:78]: chore (blocking): should bye cypress.json (lowercase)

File: README.md [hunk 3]
```diff
@@ -73,6 +73,14 @@ Run `yarn nx run <my-app>-e2e:e2e` to execute the e2e tests with [cypress](https
...
+
+1. Change `tradingTestVegaWalletName` to your Vega wallet name
```
Comment (mattrussell36) [inline README.md:80]: chore (non blocking): Can we make the casing of all cypress env variables consistent?

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jtsang586) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (jtsang586) [reply README.md]: Also done

File: README.md (hunk 3 above)
Comment (jtsang586) [reply README.md]: I've updated all the variables to snake case now. 
I kind of liked that it was different cases for the ones passed in from Github secrets but it makes sense to keep it all consistent.

**Review by mattrussell36 — APPROVED** [review]

**Review by dexturr — APPROVED** [review]

**Pull request merged**